BEIMA.Backend.Test/AuthenticationService/AuthenticationServiceTest.cs

```diff
@@ -0,0 +1,157 @@
+﻿using BEIMA.Backend.AuthService;
+using BEIMA.Backend.Models;
+using BEIMA.Backend.MongoService;
+using JWT;
+using JWT.Algorithms;
+using JWT.Serializers;
+using Microsoft.AspNetCore.Http;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static BEIMA.Backend.Test.RequestFactory;
+
+namespace BEIMA.Backend.Test.AuthenticationService
+{
+    [TestFixture]
+    public class AuthenticationServiceTest : UnitTestBase
+    {
+        [Test]
+        public void ServiceNotCreated_CallConstructorFiveTimes_FiveInstancesAreEqual()
+        {
+            //Tests to make sure singleton is working
+            var authSerivce1 = AuthService.AuthenticationService.Instance;
+            var authSerivce2 = AuthService.AuthenticationService.Instance;
+            var authSerivce3 = AuthService.AuthenticationService.Instance;
+            var authSerivce4 = AuthService.AuthenticationService.Instance;
+            var authSerivce5 = AuthService.AuthenticationService.Instance;
```

Comment:
AuthService.AuthenticationService is used because of namespace conflicts between Test.AuthenticationService
and AuthService.AuthenticationService. This only needed in the tests and the 'AuthService.' will not appear in the normal functions

Reply:
I think the issue is that your folder/namespace name is the same as your class name. If you look at the MongoConnector/MongoConnectorTest, they are in folders/namespaces called MongoService. I would suggest possibly using Authentication or AuthService as the folder names, and AuthenticationService/AuthenticationServiceTest as the class names, and then I think your namespace conflicts will be resolved.